### PR TITLE
Changed website to FINOS Branding & changes to Custom CSS & New Logos

### PR DESCRIPTION
**Pull request by grizzwolf**
For a view of the changes proposed in this PR - please refer to my test site for FDC3: https://grizzwolf.github.io/FDC3/

### CODE REVIEW

**Comment by grizzwolf** [comment]
Hi @rikoe - gotcha on those changes. Have just run through them locally.

I've changed:

- Padding on top and bottom of interior pages in the .mainContainer to be inline with what was there before. (not sure why it reverted to 0 padding - but that helps with some of the other microsites I'm working on currently too)

- changed color scheme for mobile navigation - and changed indication color to show which page you're on in the mobile nav

- added the users.json file - Chart IQ - that I've done in https://github.com/FDC3/FDC3/pull/100 to be consistent (if I missed something there, please let me know).

@rikoe - is it best for me to submit a new PR with these changes? I don't know how to change this PR (if you can even do that).

Thanks - Griz

**Comment by rikoe** [comment]
No need to do anything extra - if you add your commits to the same branch that you did the PR from, it automatically rolls forward to incorporate them, e.g. see https://github.com/FDC3/FDC3/pull/96/files/c528e24587710959f816b993617019a2fff804f7..d2dfa45b4d44a298cae2c2d2935de2b7cc46932f.

I have had a look at the changes, looks good to me now, thanks!

**Comment by rikoe** [comment]
@grizzwolf are you sure the changes you just made to the img paths worked? On your website I see this:

image

**Comment by grizzwolf** [comment]
I see that too - let me confirm.

**Comment by grizzwolf** [comment]
In the FDC3 repo - your json file has     "image": "/img/users/adaptive.webp",     with the "/" before img.

I think my set up may be different in the config file (something Mao and I were working on to get the initial test site going) - let me duplicate the siteConfig.js file from the current version to get it all on par.

**Comment by grizzwolf** [comment]
my base URL is different

**Comment by rikoe** [comment]
> my base URL is different

Why did you need to make the change to remove the leading slash in the first place, it worked fine before (and is working fine at the moment)?

File: website/siteConfig.js [hunk 1]
```diff
@@ -15,12 +15,12 @@ const users = require('./data/users.json');
 const siteConfig = {
   title: 'FDC3', // Title for your website. THIS CHANGES THE TITLE TAGS FOR THE WEBSITE (LOOK IN BROWSER TAB AT TOP)
   tagline: 'Open standards for the financial desktop', //THIS CHANGES THE TITLE TAGS FOR THE WEBSITE (LOOK IN BROWSER TAB AT TOP)
-  url: 'https://grizzwolf.github.io/FDC3/',
-  //cname: 'fdc3.finos.org',
-  baseUrl: '/FDC3/',
+  url: 'https://fdc3.finos.org',
```
Comment (rikoe) [inline website/siteConfig.js]: Ah. I see what happened now. This is a thorny problem to solve, it happened to me to. I think it is an unresolved issue on Docusaurus, how to target the same config & website at different URLs.

👍

Comment (grizzwolf) [reply website/siteConfig.js]: yes - that's better than my explanation. It was frustrating when we couldn't get it running, and figured out the base URL change (when forking in personal repos). Let me know if any issues other than that.

**Comment by grizzwolf** [comment]
@rikoe - quick thing. I think there was a logo that wasn't working in a Travis CI build on https://grizzwolf.github.io/FDC3/. I temporarily reverted the siteConfig.js and users.json files to make the Travis CI build work with all the logos - then reverted back to the correct version for the PR. Should be good to go - Gab and I just didn't want someone looking at the logos and think we missed something or something was broken. Thanks!

**Comment by grizzwolf** [comment]
@rikoe Thanks for putting up the merge! I've been checking a couple of things - and I found out that iOS doesn't support .webp images (from Firefox). Everything looks great on desktop - but on an iPhone, all but the Scott Logic "user" images are hidden. Scott Logic was the only one that was a .png.

Do you want me to re-add all of these images as .png's and then post another PR? adding @maoo to this for reference too.
Thanks - Grizz

**Comment by grizzwolf** [comment]
Plus - I have access to all the image files for the FINOS members - so I'll just do that anyway and send the PR. Looks like the standard Docusaurus Users addition is the .webp - and iOS (both Chrome and Safari) have not accepted it yet: 

https://appleinsider.com/articles/18/10/05/firefox-gets-support-for-googles-webp-image-format-leaving-apple-safari-as-main-holdout 

Mozilla this week announced that it will support Google's compressed WebP image format in Firefox, leaving Apple's Safari as the one flagship browser without any compatibility. ... Because of Apple policies, third-party iOS browsers — even Google Chrome — must use the same underlying engine as Safari.Oct 5, 2018

**Comment by rikoe** [comment]
That sounds like a great idea. Thanks for catching this.

Riko

**Comment by grizzwolf** [comment]
Roger - working on it now. (and this helps me with other Microsites down the road...) I'll also note it in the README.md

**Pull request merged**